wiki/Legal/SCTL/en.md

```diff
@@ -0,0 +1,37 @@
+---
+legal: true
+---
+
+# The following information is disclosed based on Article 11 of the "Act on Specified Commercial Transactions"
```

Comment:
using the lead paragraph as the title looks very weird

```suggestion
# SCTL

The following information is disclosed based on Article 11 of the "Act on Specified Commercial Transactions"
```

or

```suggestion
# SCTL disclosure

The following information is disclosed based on Article 11 of the "Act on Specified Commercial Transactions"
```

Reply:
As in this is the standard layout. It's important that this is the header. I don't want to mess this up due to technicalities, and Japan loves technicalities.

This page isn't for end users so doesn't have to look pretty.

Reply:
🤷